mmv1/products/netapp/volume.yaml

```diff
@@ -0,0 +1,416 @@
+              name: 'kerberos5iReadOnly'
+              description: |
+                If enabled (true) the rule defines a read only access for clients matching the 'allowedClients' specification. It enables nfs clients to mount using 'integrity' kerberos security mode.
+            - !ruby/object:Api::Type::Boolean
+              name: 'kerberos5iReadWrite'
+              description: |
+                If enabled (true) the rule defines read and write access for clients matching the 'allowedClients' specification. It enables nfs clients to mount using 'integrity' kerberos security mode. The 'kerberos5iReadOnly' value is ignored if this is enabled.
+            - !ruby/object:Api::Type::Boolean
+              name: 'kerberos5pReadOnly'
+              description: |
+                If enabled (true) the rule defines a read only access for clients matching the 'allowedClients' specification. It enables nfs clients to mount using 'privacy' kerberos security mode.
+            - !ruby/object:Api::Type::Boolean
+              name: 'kerberos5pReadWrite'
+              description: |
+                If enabled (true) the rule defines read and write access for clients matching the 'allowedClients' specification. It enables nfs clients to mount using 'privacy' kerberos security mode. The 'kerberos5pReadOnly' value is ignored if this is enabled.
+  - !ruby/object:Api::Type::Array
+    name: 'protocols'
+    description: |
+      The protocol of the volume. Allowed combinations are `['NFSV3']`, `['NFSV4']`, `['SMB']`, `['NFSV3', 'NFSV4']`, `['SMB', 'NFSV3']` and `['SMB', 'NFSV4']`.
+    required: true
+    immutable: true
+    item_type: !ruby/object:Api::Type::Enum
+      name: 'protocols'
+      description: |
+        Access protocol types.
+      values:
+        - :NFSV3
+        - :NFSV4
+        - :SMB
+  - !ruby/object:Api::Type::Array
+    name: 'smbSettings'
+    description: |
+      Settings for volumes with SMB access.
+    item_type: !ruby/object:Api::Type::Enum
+      name: 'smbSettings'
+      description: |
+        Settings for volumes with SMB access.
+      values:
+        - :ENCRYPT_DATA
+        - :BROWSABLE
+        - :CHANGE_NOTIFY
+        - :NON_BROWSABLE
+        - :OPLOCKS
+        - :SHOW_SNAPSHOT
+        - :SHOW_PREVIOUS_VERSIONS
+        - :ACCESS_BASED_ENUMERATION
+        - :CONTINUOUSLY_AVAILABLE
+  - !ruby/object:Api::Type::String
+    name: 'unixPermissions'
+    description: |
+      Unix permission the mount point will be created with. Default is 0770. Applicable for UNIX security style volumes only.
+    default_from_api: true
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      Labels as key value pairs. Example: `{ "owner": "Bob", "department": "finance", "purpose": "testing" }`.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      An optional description of this resource.
+  # We want to get rid of this. Please don't expose.
```

Comment:
Remove this if it's deprecated

Reply:
This is currently in discussion. As you can see, 'snapReserve' is already commented out. So while it is still in the yaml file, is it not exposed to users.